### ISSUE TITLE
curvy vertical prisms?

### ISSUE THREAD

**Issue by jaredjennings** [jaredjennings/dactyl-keyboard#42]
The vertical prisms yield pieces with points and edges. These tend to be exaggerated by the 3D printer, leading to surfaces that contact each other only poorly, and are difficult to improve by sanding.

Try adding more polys to the vertical prisms, to make them curvier; see if this yields a better fit.

**Comment by jaredjennings** [comment]
Also if you have more lace holes (#43) you have more places where the orderly flat and angled vertical prisms are interrupted

**Comment by jaredjennings** [comment]
Ok, hacking the hexagonal vertical prisms into sinusoidal ones worked initially, but then making those bumps appear where I wanted on the half-circle connectors (#44) proved to be very difficult, and involved factors and addends that, for example, changed when I changed the period of the sines, or when I changed the resolution of the thing. And it was slow. So I'm undoing the whole thing, reopening this issue, and I'm going to do the work in a branch.
